Source/CombatExtended/CombatExtended/PatchOperationMakeGunCECompatible.cs

```diff
@@ -182,6 +182,10 @@ private void AddOrReplaceVerbPropertiesCE(XmlDocument xml, XmlNode xmlNode)
             {
                 // remove Verb_Shoot
                 var verb_shoot_nodes = verbs.SelectNodes("li[verbClass=\"Verb_Shoot\" or verbClass=\"Verb_ShootOneUse\" or verbClass=\"Verb_LaunchProjectile\"]");
+                if (ModLister.HasActiveModWithName("Vanilla Weapons Expanded - Makeshift"))
+                {
+                    verb_shoot_nodes = verbs.SelectNodes("li[verbClass=\"Verb_Shoot\" or verbClass=\"Verb_ShootOneUse\" or verbClass=\"Verb_LaunchProjectile\" or verbClass=\"VWEMakeshift.Verb_MakeshiftShoot\"]");
+                }
```

Comment:
There has to be a better way to do this... If another mod has a unique verbclass there will be a conflict...

Reply:
Agreed. One example is VFE's `SmokingGun.Verb_ShootWithSmoke`.

Reply:
I guess just add onto the list...

Reply:
Well, um, `SmokingGun.Verb_ShootWithSmoke` is actually from the base Vanilla Expanded Framework itself, rather than any of its spinoffs.

I say this because I'm working on a non-Vanilla Expanded mod that uses VFE as a dependency, and I'm certain there are other non VE mods that do so as well. So we'd either have to add a new `if (ModLister.HasActiveModWithName("Vanilla Expanded Framework"))` block, or come up with a more elegant solution as you originally suggested.

Reply:
For now we can increase this list without consequence but have to be mindful of other mods like the SmokingGun example when creating patches for CE

Reply:
@sumghai I guess after this PR add the custom verb to the string for SelectNodes?